scripts/styleguide.config.js

```diff
@@ -270,21 +276,89 @@ module.exports = {
     styles: {
         Heading: {
             heading: {
-                lineHeight: 1,
+                lineHeight: 2,
+            },
+        },
+        Pathline: {
+            pathline: {
+                background: vars['bdl-box-blue-05'],
+                borderRadius: 3,
+                color: vars['bdl-gray'],
+                display: 'inline',
+                fontSize: 12,
+                padding: 10,
+            },
+        },
+        Code: {
+            code: {
+                fontSize: 12,
```

Comment:
It looks like this is set quite a few times. Is there a more central option, such as `theme.fontSize.base` or `theme.fontSize.small`? I'm not certain if the theme affects the layout-level elements.

Reply:
They use the base fontSize which is too large with the default monospace font